modules/aws-vpn-vpnendpoint/variables.tf

```diff
@@ -90,6 +91,6 @@ variable "vpn_port" {
 }
 
 variable "dns_server" {
```

Comment:
better to set name to plural "dns_servers" as now it can be multiple ones,
and also have some minimal description on this